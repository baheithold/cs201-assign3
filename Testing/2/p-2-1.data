3071 638 143 ; 6353 4558 233 ; 4162 3246 364 ; 2302 2761 361 ; 404 5141 327 ; 
7465 7379 205 ; 7541 8945 153 ; 9756 8672 456 ; 6998 7417 185 ; 4763 40 63 ; 

