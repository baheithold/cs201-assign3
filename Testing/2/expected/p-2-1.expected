0: 3071
1: 638(3071)143
weight: 143
